ext/js/pages/info-main.js

```diff
@@ -154,6 +155,17 @@ await Application.main(true, async (application) => {
     const settingsController = new SettingsController(application);
     await settingsController.prepare();
 
+    /** @type {ThemeController} */
+    const themeController = new ThemeController(document.documentElement);
+    themeController.prepare();
+    const optionsFull = await application.api.optionsGetFull();
+    const {profiles, profileCurrent} = optionsFull;
+    const primaryProfile = (profileCurrent >= 0 && profileCurrent < profiles.length) ? profiles[profileCurrent] : null;
```

Comment:
```suggestion
    const primaryProfile = (0 <= profileCurrent && profileCurrent < profiles.length) ? profiles[profileCurrent] : null;
```

https://llewellynfalco.blogspot.com/2016/02/dont-use-greater-than-sign-in.html